apps/src/applab/AppLabView.jsx

```diff
@@ -206,7 +206,7 @@ var AppLabView = React.createClass({
 module.exports = connect(function propsFromStore(state) {
   return {
     isReadOnlyWorkspace: state.level.isReadOnlyWorkspace,
-    instructionsInTopPane: state.level.instructionsInTopPane,
+    instructionsInTopPane: state.level.instructionsInTopPane && !!state.level.instructionsMarkdown,
```

Comment:
I started out doing it in the component itself, but there were multiple places I needed to add the check, and I wasn't sure I was getting them all.

I hadn't thought about putting this in this initialize, but that might make sense.

I wonder if what actually would make the most sense would be to have the prop for AppLabView be different, i.e. in our connect we have `showInstructions: state.level.instructionsInTopPane && !!state.level.instructionsMarkdown`. Now this concept is more clearly distinct.